oat-sa/extension-tao-testqti
# Feature/INV-173/prettify XML code editor

**Related to task:** 
- https://oat-sa.atlassian.net/browse/INV-170
- https://oat-sa.atlassian.net/browse/INV-173

**Description:** Highlight XML code
<img width="921" alt="Screenshot 2020-06-18 at 11 20 50" src="https://user-images.githubusercontent.com/6339002/85002916-c98cb600-b155-11ea-989a-813efe36191c.png">
**Requires:** 
- [ ] https://github.com/oat-sa/tao-core/pull/2558
- [ ] https://github.com/oat-sa/grunt-tao-bundle/pull/16 for production build 

**How to test:** 
- Create user with "Test XML Editor" role
- In `config/taoQtiTest/XmlEditor.conf.php` set the option `is_locked` to `false`
- Navigate to the QTI test page and click on "XML Editor" 
- The textarea should be replaced with [codemirror](https://codemirror.net/) editor
